Cargo.toml

```
@@ -82,7 +86,9 @@ ethereum_ssz_derive = "0.9"
 eventsource-client = "0.15.0"
 futures = "0.3"
 hashbrown = "0.15.3"
-hashsig = { git = "https://github.com/b-wagn/hash-sig" }
+hashsig = { git = "https://github.com/b-wagn/hash-sig.git" }
```

```suggestion
hashsig = { git = "https://github.com/b-wagn/hash-sig.git", rev = "287517a763edba7e518b0c1ee5beb868f26f1f66" }
```

How about lock the version, as it is under development actively?